src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -171,6 +178,7 @@ export function TravellerSelection({
       <Section {...accessibility}>
         <GenericClickableSectionItem
           onPress={travellerSelectionOnPress}
+          disabled={!canSelectUserProfile}
```

Comment:
I think we shouldn't use a disabled button here. I know its faster to implement this way and less code, but semantically not correct.

Reply:
We can leave it clickable, but then it's a bit weird when you can click a button but nothing shows up. Another way is probably switch to another component which are not clickable if they can't select it. Which one do you think is better? I'm open to suggestions and I'll make another PR to update it. 😄

Reply:
Ment more like dragging out the stuff inside `GenericClickableSectionItem` to be reusable, and then something like this:
```
canSelectUserProfile 
  ? <GenericClickableSectionItem ...>...</GenericClickableSectionItem>
  : <GenericSectionItem ...>...</GenericSectionItem>
```

Reply:
Yes that is what i meant by "switch to other component when it's not clickable" 👍

Reply:
I'm making a separate PR for this one :)

Reply:
https://github.com/AtB-AS/mittatb-app/pull/4364